thefiletree.docker

```diff
@@ -0,0 +1,26 @@
+# Take the latest version of nodejs container.
+FROM google/nodejs
+MAINTAINER David Aparicio "david.aparicio@free.fr"
+
+# Install tree build dependencies.
```

Comment:
Nit: To be consistent with below, please use `Tree` instead of `tree`. `tree` is an internal name, and we should capitalize it outside of its repositories to not confuse people.